### PR TITLE
Fixed SMA spline creation function

### PR DESCRIPTION
**Pull request by BirchellE**
- Fixed sorting bug in market data controller
- Changed createIndicatorSeries to work with old way of adding series
- Fixed function for adding SMA splines

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -216,24 +217,45 @@ $(document).ready(() => {
...
-    console.log(newIndObj);
-    console.log(mainChart);
-    mainChart.addSeries(newIndObj);
-    
+    console.log(chartData);
+
+    // Redrawing chart after adding indicators
+    mainChart.redraw();
+
+    //console.log(newIndObj);
```
Comment (adamlawrencium) [inline public/js/main.js:255]: delete commented out things that we probably won't use again

Comment (BirchellE) [reply public/js/main.js:255]: I didn't want to delete your code

**Comment by adamlawrencium** [comment]
lets use airbnb javascript linting

File: public/js/main.js [hunk 2]
```diff
@@ -216,24 +217,45 @@ $(document).ready(() => {
...
+      mainChart = Highcharts.stockChart('hcharts-strategy', createHighChartsObj(ops));
+      //$('#hcharts-strategy').highcharts('StockChart', createHighChartsObj(ops));
     });
   });
   $('#addIndicatorBtn_SMA').click(() => {
     console.log('### Inside addIndicator_SMA');
     const SMAParam = $('#addIndicator_SMA').val();
+
+    // Parsing pattern for csv SMA values
+    var pattern = /\s*,\s*/;
```
Comment (aeksco) [inline public/js/main.js:232]: Try to avoid `var` when possible - best to use either `let` or `const` going forward

File: public/js/main.js (hunk 1 above)
Comment (adamlawrencium) [reply public/js/main.js:255]: if you're replacing a feature/implementation, you can delete the code associated with that feature, unless we'll need it in the future.

**Pull request merged**